src/test/resources/mail/email-change-confirmation.html

```
@@ -0,0 +1,3 @@
+{{username}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)